.drone.yml

```diff
@@ -1003,6 +1003,91 @@ services:
       - name: dockersock
         path: /var/run
 
+volumes:
+  - name: dockersock
+    temp: {}
+---
+kind: pipeline
+type: kubernetes
+name: teleport-docker-fpm
+
+trigger:
+  cron:
+    - teleport-docker-fpm
+  repo:
+    include:
+      - gravitational/teleport
+
+workspace:
+  path: /go
+
+clone:
+  disable: false
+
+steps:
+  - name: Wait for docker
+    image: docker
+    volumes:
+      - name: dockersock
+        path: /var/run
+    commands:
+    - timeout 15s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
+
+  - name: Set up Dockerfile
+    image: docker:git
+    environment:
+    commands:
+      - apk --update --no-cache add curl
+      - mkdir -p /go/build
+      # get Dockerfiles
+      - curl -Ls -o /go/build/Dockerfile-fpm https://raw.githubusercontent.com/gravitational/teleport/${DRONE_SOURCE_BRANCH:-logan/add-fpm-drone}/build.assets/Dockerfile-fpm
```

Comment:
Presumably you're going to change this reference to `master` and resign before merge? :)